docs/manage/_content.rst

```diff
@@ -66,30 +71,32 @@ If the computer where Kolibri is running has an Internet connection with the suf
 Import Content from a Local Drive
 *********************************
 
-If the computer where Kolibri server is running does not have access to Internet or has insufficient bandwidth, you have the option to receive content channels stored on an external drive (USB stick or hard disk). Follow these steps to import content channels. 
+If the computer where Kolibri server is running does not have access to Internet or has insufficient bandwidth, you have the option to receive content channels stored on an external drive (USB stick or hard disk). Follow these steps to import content channels.
 
-#. Insert the USB drive in your computer.
+#. Connect the external USB drive to your computer.
 #. Choose option for *Attached drive or memory card*, and click **Continue**.
 #. Kolibri will automatically detect and display the drive(s) with available Kolibri content files.
 #. Select the drive where the desired channel is stored, and click **Continue**.
 #. Click **Select** button for the desired channel, and follow the same steps for selecting topics and resources as for the :ref:`import from Kolibri Central Server <central_server>`. 
 
-	.. image:: img/import-local-drive2.png
-	  :alt: import channel from detected local drive
+  .. image:: img/import-local-drive2.png
+    :alt: import channel from detected local drive
 
 
-.. tip:: **Workaround for import from local drive on older devices.**
+.. tip:: **Workaround for import from external drive on older devices.**
```

Comment:
'local' seems more accurate than 'external'. what was the goal of this change?

Reply:
In my mind `local` does not render the idea of the drive being "brought from elsewhere", which is the primary use case for this set of instructions. Local HD may as well be understood to be the main HD on the device running Kolibri, and I felt the distinction needed to be stronger.

Reply:
What ever you feel is most appropriate.

Just noting that some people have multiple local internal drives or partitions.

Reply:
Inconsistent terminology detected! @radinamatic could you make a note or issue that we should decide whether to call it external or local drive? It's kind of mixed up :)